transformer_engine/common/fused_attn/fused_attn_f16_max512_seqlen.cu

```diff
@@ -4,7 +4,7 @@
  * See LICENSE for license information.
  ************************************************************************/
 
-#include "fused_attn_fp16_bf16_max_seqlen_512.h"
+#include "fused_attn_f16_max512_seqlen.h"
```

Comment:
Does a change in both of these file names imply dropped bf16 support?

Reply:
No, this is only for brevity. f16 means all float 16: bf16 and fp16.

Reply:
Oops my bad, I read the filename as fp16 and not f16